src/Microsoft.Azure.WebJobs.Host/Executors/JobHostContext.cs

```diff
@@ -78,7 +78,7 @@ public IAsyncCollector<FunctionInstanceLogEntry> FastLogger
 
         public void Dispose()
         {
-            if (_disposed)
```

Comment:
Doh! This error has been in the SDK since the early days (2014 at least: https://github.com/Azure/azure-webjobs-sdk/blob/95c5f4547749a89f69ee3a12c9c252fd3af51144/src/Microsoft.Azure.WebJobs.Host/Executors/JobHostContext.cs)